package.json

```diff
@@ -9,7 +9,7 @@
     "url": "git://github.com/marmelab/ng-admin.git"
   },
   "devDependencies": {
-    "admin-config": "^0.11.0",
+    "admin-config": "^0.11.1",
```

Comment:
I have released 0.12.0

As a side note, 0.0.x is for bugfixes, 0.x is for new features. `.sortable()` is a new feature. That's called semver.